ehb_service/apps/api/views/externalsystem.py

```diff
@@ -184,11 +180,11 @@ def get(self, request, pk):
             es = ExternalSystem.objects.get(pk=pk)
         except ExternalSystem.DoesNotExist:
             log.error("ExternalSystem[{0}] not found".format(pk))
-            return HttpResponse(status=codes.not_found)
+            return Response(status=status.HTTP_404_NOT_FOUND)
 
         r = es.responseFieldDict()
-
-        return json.dumps(r)
+        r = json.loads(json.dumps(r))
```

Comment:
Why do we need to do a json.dumps then a json.loads?

Reply:
forgot to remove this step! removed in this commit  e396c62